packages/frontend/src/components/BidsList/BidsListSection.tsx

```diff
@@ -70,11 +70,15 @@ function selectBids(
   const lastAuctionBidIndex = bids.length > auctionWinnersCount ? auctionWinnersCount - 1 : bids.length - 1
   const lastAuctionBid = bids[lastAuctionBidIndex]
 
-  return userBid && within(bidsMaxCount, auctionWinnersCount - 1, userBid.place)
+  return userBid && checkIfUserBidIsNotLastOne(userBid, lastAuctionBid, auctionWinnersCount)
     ? topAuctionBids.concat([userBid, lastAuctionBid])
     : topAuctionBids.concat([lastAuctionBid])
 }
 
+const checkIfUserBidIsNotLastOne = (userBid: UserBid, lastAuctionBid: Bid, auctionWinnersCount: number) => {
```

Comment:
Let's rename that function as it doesn't only check if user's bid is the last one, but also if bid is within given range. Maybe something like `shouldUserBidBeDisplayed` or something more general?